src/main/webapp/i18n/en/exercise-actions.json

```
@@ -66,6 +66,7 @@
             "uploadFile": "Upload a file",
             "viewTeam": "Team",
             "sshKeyTip": "To use ssh, you need to add an ssh key to your account {link:here}.",
+            "sshKeyExpiredTip": "Some of your SSH keys are expired. You might need to a new ssh key to your account {link:here}.",
```

Do we want to use `SSH` or `ssh`?

```suggestion
            "sshKeyTip": "To use SSH, you need to add an SSH key to your account {link:here}.",
            "sshKeyExpiredTip": "Some of your SSH keys are expired. You might need to a new SSH key to your account {link:here}.",
```